rb/spec/integration/selenium/webdriver/spec_helper.rb

```diff
@@ -17,14 +17,6 @@
 # specific language governing permissions and limitations
 # under the License.
 
-begin
```

Comment:
Just pushed https://github.com/SeleniumHQ/selenium/commit/7e198932c15a8b0f8ef672a49acf82025603d22c so there should be no more conflicts.